addons/block_code/block_code_node/block_code.gd

```diff
@@ -36,22 +29,18 @@ func _enter_tree():
 		plugin.add_inspector_plugin(load("res://addons/block_code/inspector_plugin/block_script_inspector.gd").new())
 
 
-# Necessary to "export" the block script data without exposing it
-func _get_property_list():
```

Comment:
Yea, maybe just comment out this function instead of removing it if you want, up to you. Not sure what the standard is.

Reply:
Actually it's a bit hard because it involves setting `PROPERTY_USAGE_EDITOR` or `PROPERTY_USAGE_NO_EDITOR` instead of the usual `@export`. I don't think we should hide this, at least for the time being.